src/components/Menu/MenuItemAnchor.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactElement } from 'react';
+import styled from 'styled-components';
+
+interface MenuItemAnchorProps {
+  link: string;
+  title: string;
+  icon?: ReactElement;
+}
+
+const MenuItem = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin-bottom: 10px;
+  margin-right: 12px;
+  cursor: pointer;
+
+  a,
+  a:visited {
+    color: ${({ theme }) => theme.color.text.topMenu};
+    text-decoration: none;
+  }
+
+  a:hover {
+    color: ${({ theme }) => theme.color.text.settingsIcon};
+    text-decoration: none;
+  }
+
+  &:hover {
+    color: ${({ theme }) => theme.color.text.settingsIcon};
+    text-decoration: none;
+    #etherspot-logo {
```

Comment:
can u wrap image as `styled-components` and include into this as `${EtherspotLogp} { someCss }`

Reply:
Working on this!

Reply:
@poocart Updated the code!